src/main/host/host.c

```diff
@@ -116,7 +120,7 @@ static SimulationTime _maxUnappliedCpuLatencyConfig;
 ADD_CONFIG_HANDLER(config_getMaxUnappliedCpuLatency, _maxUnappliedCpuLatencyConfig)
 
 /* this function is called by manager before the workers exist */
-Host* host_new(const HostParameters* params) {
+Host* host_new(const HostParameters* params, EmulatedTime simEndTime) {
```

Comment:
Is there a reason to make this a separate parameter instead of adding to HostParameters?

Reply:
Not really. I've moved this into `HostParameters`.